schemas/test/en/test_list_collector_repeating_blocks.json

```diff
@@ -0,0 +1,227 @@
+                                    {
+                                        "id": "list-collector-next-question-answer-1",
+                                        "mandatory": true,
+                                        "type": "Radio",
+                                        "options": [
+                                            {
+                                                "label": "Yes",
+                                                "value": "Yes",
+                                                "action": {
+                                                    "type": "RedirectToListAddBlock"
+                                                }
+                                            },
+                                            {
+                                                "label": "No",
+                                                "value": "No"
+                                            }
+                                        ]
+                                    }
+                                ]
+                            },
+                            "add_block": {
+                                "id": "list-collector-add-block",
+                                "type": "ListAddQuestion",
+                                "question": {
+                                    "id": "list-collector-add-block-question",
+                                    "type": "General",
+                                    "title": "What is the name of the list item? This item will be followed up by 2 pages of related questions (repeating blocks)",
+                                    "answers": [
+                                        {
+                                            "id": "list-collector-add-block-question-answer",
+                                            "label": "Name of list item",
+                                            "mandatory": true,
+                                            "type": "TextField"
+                                        }
+                                    ]
+                                }
+                            },
+                            "repeating_blocks": [
+                                {
+                                    "id": "repeating-block-1",
+                                    "type": "ListRepeatingQuestion",
+                                    "question": {
+                                        "id": "repeating-block-1-question",
+                                        "type": "General",
+                                        "title": "This is the question for repeating block 1, which relates to an item that was added to the list collector.",
+                                        "answers": [
+                                            {
+                                                "id": "repeating-block-1-question-answer-1",
+                                                "label": "Answer 1",
```

Comment:
Should have some description otherwise you hit validation errors cause you don't know what answer type is expected.

Reply:
Sorted 👍